controller/src/test/java/io/pravega/controller/server/SegmentHelperTest.java

```diff
@@ -713,7 +715,10 @@ public Host getHostForTableSegment(String table) {
 
         @Override
         public ScheduledExecutorService getInternalExecutor() {
-            return null;
+            if (executor == null) {
```

Comment:
I don't recommend using this pattern. Please set the executor either in a constructor or when you declare the field.

Reply:
Got it.